lib/manager/gomod/artifacts.spec.ts

```diff
@@ -96,14 +93,15 @@ describe('manager/gomod/artifacts', () => {
     ).toBeNull();
     expect(execSnapshots).toMatchSnapshot();
   });
+
   it('returns updated go.sum', async () => {
-    fs.readFile.mockResolvedValueOnce('Current go.sum' as any);
-    fs.readFile.mockResolvedValueOnce(null as any); // vendor modules filename
+    fs.readLocalFile.mockResolvedValueOnce('Current go.sum');
+    fs.readLocalFile.mockResolvedValueOnce(null as any); // vendor modules filename
```

Comment:
```suggestion
    fs.readLocalFile.mockResolvedValueOnce(null); // vendor modules filename
```

You missed some 😕

Reply:
Hi @viceice sorry, I didn't have a chance to fix this yet, since I struggle with `gradle-wrapper` right now.